core/src/main/scala/kafka/coordinator/group/GroupMetadata.scala

```diff
@@ -144,10 +144,9 @@ private object GroupMetadata extends Logging {
     members.foreach(member => {
```

Comment:
nit: This is not related to your patch but could we replace `foreach(member => {` by `foreach { member =>`?